### DISCUSSION TITLE
Einstellungen für SmartPhone Layout

### DISCUSSION THREAD

**Discussion by mcuiobroker** [Zefau/ioBroker.jarvis#2121] [Q&A]
Kann man die internen Einstellungen für das SmartPhone Layout irgendwie selbst beeinflussen?
Hintergrund
https://forum.iobroker.net/topic/49776/jarvis-v3-0-0-just-another-remarkable-vis/3559
Tablet wird auch als "SmartPhone" erkannt. Dies scheint aber nichts mit Auflösung zu tun zu haben.?
Einstellungen bzgl. Theme greifen nicht.

**Comment by Zefau** [comment]
Nein, aber mach gerne einen Feature Request auf. Ich hätte ja mal Daten gesammelt, um besser zwischen Smartphone und Tablet zu unterscheiden, habe aber damals nichts weiter implementiert.

**Comment by git2206** [comment]
ok, kann ich gerne machen... nur wird das dann erst mit version 3.2 implementiert oder schon in 3.0.x...? 

mfg
Flo

**Reply by Zefau** [reply]
Features laufen jetzt in die v3.2. v3.1 und v3.0 werden nicht mehr mit Features versorgt.
